src/main/java/frc/robot/RobotContainer.java

```diff
@@ -29,6 +37,15 @@ private void configureButtonBindings() {
 
     // Turret Commands
     new JoystickButton(m_driverController, 1).whileHeld(new TurretJoystickCommand(0.3, m_turret));
+
+    // Intake Commands
+    new JoystickButton(m_driverController, 2).whileHeld(new RunIntake(m_intake, 0.5));
+    new JoystickButton(m_operatorController, 1).whenPressed(new ToggleIntake(m_intake));
+    new JoystickButton(m_operatorController, 3).whileHeld(new ToggleCompressor(m_intake));
```

Comment:
Toggle şeklindeki bir commandi whileHeld() ile kullanmak zorluk çıkarabilir. whenPressed() daha iyi olabilir.

Reply:
Onu whenPressed yapmıştım ama ikinciye yazarken unutmuşum düzelttim :D

Reply:
düzeltildi.

Reply:
nerde @OVEYMTarik57 ?

Reply:
@vatanaksoytezer düzelti yi düzelttiklerim için yazdım kod yazarken karışmasın diye. Bir kaç dakika önce attım.